packages/verify-contract/src/common/url.ts

```diff
@@ -77,6 +77,8 @@ const DEFAULT_SCAN_API_URLS: Map<NetworkName, string> = new Map([
     ['kava', 'https://kavascan.com/api'],
     ['kava-mainnet', 'https://kavascan.com/api'],
     ['kava-testnet', 'https://testnet.kavascan.com/api'],
+    ['klaytn', 'https://api-baobab.klaytnscope.com/api'],
+    ['klaytn-mainnet', 'https://api-baobab.klaytnscope.com/api'],
```

Comment:
this may be testnet...looking into this

Reply:
yup -- mainnet api url is "https://api-cypress.klaytnscope.com/api", will update here soon